server/.gitignore

```diff
@@ -0,0 +1,4 @@
+ui-server
+api
+openapi/assets/temporal
+ui/assets
```

Comment:
if UI assets are ignored will we still be able to serve the UI when /server is consumed as a module? e.g from Temporal CLI